webapp/components/wpt-metadata.js

```diff
@@ -100,7 +100,7 @@ class WPTMetadata extends PathInfo(LoadingState(PolymerElement)) {
               ></wpt-metadata-node>
             </template>
           </iron-collapse>
-          <paper-button id="metadata-toggle" onclick="[[openCollapsible]]">
```

Comment:
nit: this variable should be removed in the file.

Reply:
Done.